pkg/util/logger.go

```diff
@@ -12,7 +12,6 @@ import (
 	"github.com/go-kit/kit/log"
```

Comment:
This whole file should be deleted. If anything is needed it should go in the pkg/logging